### DISCUSSION TITLE
What can I do if I have lost my 2FA creds and also lost the recovery codes?

### DISCUSSION THREAD

**Discussion by eliran-natan** [community/community#37995] [Other Feature Feedback, Questions, & Ideas]
Please help me to restore my GitHub account. I have lost both my 2FA creds and also lost the recovery codes.  thanks.

**Comment by radar-blip** [comment]
For account related issues, see here: [Support page](https://support.github.com/contact/account)

Good luck!

**Reply by Nakin012519Dathong** [reply]
ผมจะใช้เงินได้อย่างไร
